src/node.js

```diff
@@ -388,17 +389,17 @@
       tickDone();
     }
 
+    function PObj(c) {
+      this.callback = c;
+      this.domain = process.domain || null;
+    }
+
     function nextTick(callback) {
       // on the way out, don't bother. it won't get fired anyway.
       if (process._exiting)
         return;
 
-      var obj = {
-        callback: callback,
-        domain: process.domain || null
-      };
-
-      nextTickQueue.push(obj);
+      nextTickQueue.push(new PObj(callback));
```

Comment:
@benjamingr actually use of `new PObj()` is indeed faster. I thought the same thing, but numbers don't lie. :)